doc/faq.rst

```diff
@@ -22,25 +22,39 @@ need to install the ``virtualenv`` package, and then create a new virtual enviro
     virtualenv --python=python3 ~/eos
 
 
-The new virtual environment will now be activated.
+The new virtual environment will now be **activated**.
+That means, the `python` (and `pip`) command will now execute the local binary instead of a system-wide one.
```

Comment:
[... the local binary] contained within the virtual environment, instead of the binaries installed system-wide.

Reply:
Okay!